outorg.el

```diff
@@ -413,6 +413,11 @@ is removed before converting back from Org to source-code."
   :group 'outorg
   :type 'boolean)
 
+(defcustom outorg-ensure-blank-line nil
+  "Non-nil means insert an overlay between text and blocks.
```

Comment:
I think the docstring should focus on the feature as presented to the user rather than its underlying implementation.  So, e.g. it need not mention that it uses overlays.  Instead, it could simply be something like:

```
Always show a blank line before a source code block when editing.
The blank line is not added to the source buffer after editing.
```

Reply:
I agree. That sounds good.